README.md

```diff
@@ -50,33 +50,13 @@ For command line help, visit our wiki page on Bitters’ [command line interface
   @import "base/base";
   ```
 
-5. When using Neat, uncomment the following line in `_base.scss`:
-
-  ```scss
-  @import "grid-settings";
-  ```
-
-  And import Neat after Bitters in your `application.css.scss` or main manifest file:
-
-  ```scss
-  @import "bourbon";
-  @import "base/base";
-  @import "neat";
-
-  // All other imports
-  ```
-
-  **Note:** If you are using Bitters and Neat without Ruby on Rails, you need to change the import for `neat-helpers` inside `_grid-settings.scss` to `"../neat/neat-helpers"`.
-
-  If you want to use Neat functions in Bitters, you can `@import "grid-settings";` before Neat, remove `@import "grid-settings";` from `_base.scss` and import the rest of bitters after. For example:
+5. Once Bourbon and Bitters are set up, you can begin to import your styles below them.
```

Comment:
It's got a 1 - 5


<img width="98" alt="screen shot 2016-02-29 at 5 56 56 pm" src="https://cloud.githubusercontent.com/assets/2489247/13411925/db50dc32-df0d-11e5-9109-78cf77ed128c.png">